pyfolio/tests/test_perf_attrib.py

```diff
@@ -257,6 +263,30 @@ def test_perf_attrib_regression(self):
             exposures_summary['Average Risk Factor Exposure']
         )
 
+        cumulative_returns_by_factor = pd.Series(
+            [ep.cum_returns_final(perf_attrib_output[c])
```

Comment:
Is there any way we could write a test for these things using a dataset small enough that we can test against explicit expected values? Otherwise this is kind of just using the same logic in the code and the test, hence the bug

Reply:
Yup definitely agree. I added a simpler test for these in `test_perf_attrib_simple`, where all the returns are specific returns, so these values should all be 0. Let me know if I should add more test coverage here